.circleci/config.yml

```diff
@@ -1,6 +1,9 @@
 version: 2.1
 
 parameters:
+  ci_builder_image:
+    type: string
+    default: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.47.0
```

Comment:
I pushed a commit to the monorepo to fix the build so it includes `just`, will need to do a release tho before we can use it, don't want to block on that